src/olympia/amo/utils.py

```diff
@@ -48,6 +49,7 @@
 from olympia.translations.models import Translation
 from olympia.users.models import UserNotification
 from olympia.users.utils import UnsubscribeCode
+from olympia.api.paginator import ESPaginator
```

Comment:
Please also move `ESpaginator` outside `olympia.api.paginator` since it's not API-specific anymore. 

I suggest moving it to `olympia/amo/paginator.py`, renaming `olympia/api/paginator.py` (which would only contain the DRF pagination classes) to `olympia/api/pagination.py` to keep with DRF naming.

That also means moving the tests that are specific to the paginator in `olympia/amo/tests_paginator.py` and renaming the ones concerning DRF pagination to `olympia/api/tests/test_pagination.py`.

See https://github.com/diox/olympia/commit/fe43764fbc7f9316f6fbe7b65beb1e4dd007831c for inspiration (note how I also renamed the `Paginator` class in `src/olympia/amo/templatetags/jinja_helpers.py` to avoid confusion)